packages/components/src/Menu/Menu.tsx

```diff
@@ -12,12 +11,17 @@ import classnames from "classnames";
 import { AnimatePresence, motion } from "framer-motion";
 import { useOnKeyDown } from "@jobber/hooks/useOnKeyDown";
 import { useRefocusOnActivator } from "@jobber/hooks/useRefocusOnActivator";
+import { useWindowDimensions } from "@jobber/hooks/useWindowDimensions";
 import { IconNames } from "@jobber/design";
-import { useSafeLayoutEffect } from "@jobber/hooks/useSafeLayoutEffect";
+import { usePopper } from "react-popper";
+import { useIsMounted } from "@jobber/hooks/useIsMounted";
+import ReactDOM from "react-dom";
+import { useFocusTrap } from "@jobber/hooks/useFocusTrap";
 import styles from "./Menu.css";
 import { Button } from "../Button";
 import { Typography } from "../Typography";
 import { Icon } from "../Icon";
+import { focusAttribute } from "../FormField/hooks/useFormFieldFocus";
 
 const variation = {
```

Comment:
I can't tell if this is still needed or not

I've tested out the show/hide animation on different screen sizes and it looks to be behaving identically to current master but there's definitely some things in here that no longer exist


```
  startOrStop: (position: string) => {
    let y = 10;

    if (position === "below") y *= -1;
    if (window.innerWidth < 640) y = 150;
```

that's looking for a `position === "below"` and I removed the `custom={position}` which I figure must be how it gets the `position.vertical` which would be `below` or `above` so I thought something wold break but it hasn't

we do get back a below/above from popper's `state` if this is still relevant. gonna look into it a bit more.

Reply:
ah this doesn't currently work on master

<img width="650" alt="image" src="https://github.com/GetJobber/atlantis/assets/11843215/3ca2d9c2-6d3e-4975-b496-28018f9205f1">


it used to only be a string, and this comparison might pass or fail - but then it was changed to what it is now and it's an object, so it'll never be the string "below"

https://github.com/GetJobber/atlantis/commit/4ba4981c90c998ff20fa63971b9721fdf54c54ab#diff-9912300449143379721456414d95c1dcf3414a8f0fb096d4d641b95ec0382f41L55-R63

Reply:
the `y` offset doesn't depend on the `position` at all. that's just looking at the window width, and from what I can tell it is telling the animation where to start and end

with a smaller screen we want it to move further for the animation. that will still work with this new setup since popper is only responsible for the resting position, not the animation!

Reply:
@ZakaryH isn't this function checking which viewport edge the activator is closest to, so that it can position itself above, below, right, or left, accordingly?

Reply:
@chris-at-jobber negative. that logic was all part of the `setPosition` code. instead of us doing that ourselves, we're delegating that to Popper :)

though the result of that code would set `position` to be something like `{ vertical: "below", horizontal: "right" }` and that `position` would be passed to this `startOrStop(position: string)`

as noted, that is currently not working because the comparison would always fail. however, back when it was working - this function would potentially do 2 things

- if it's "below" flip the offset so that the starting/ending position of the menu's animation is the opposite, much like you were asking about on another comment
- if the screen is below 640px, then make the ending offset position 50px greater, not entirely sure why - if we wanted a higher speed (assuming it takes the same time) or just like how it looks moving a greater distance?